packages/blockchain/algo/src/lib/services/algo.tx.ts

```diff
@@ -505,11 +540,55 @@ export const algoTx = (args: { algoWeb: AlgoWeb }) => {
          * @returns transaction id of the transaction in the blockchain
          */
         signedTransaction: async (body: TransferAlgo | TransferAlgoKMS, testnet = false, provider?: string) =>
-          AlgorandService.algoandBroadcast({
+          AlgorandService.algorandBroadcast({
             txData: await prepareSignedTransaction(body, testnet, args.algoWeb, provider),
             ...(isTransferAlgoKMS(body) && { signatureId: body.signatureId }),
           }),
       },
     },
+
+    virtualAccount: {
+      send: {
+        /**
+         * Send ALGO transaction from Tatum Ledger account to the blockchain. This method broadcasts signed transaction to the blockchain.
+         * This operation is irreversible.
+         * @param body content of the transaction to broadcast
+         * @param testnet if the algorand node is testnet or not
+         * @param provider url of the algorand server endpoint for purestake.io restapi
+         * @returns transaction id of the transaction in the blockchain or id of the withdrawal, if it was not cancelled automatically
+         */
+        signedTransaction: async (
+          body: ApiTransferAlgo | ApiTransferAlgoKMS,
+          testnet = false,
+          provider?: string,
+        ): Promise<SendOffchainResponse> => {
+          const txData = await prepareSignedTransaction(body, testnet, args.algoWeb, provider)
+
+          const { fee, ...withdrawal } = body
+
+          const { id } = await ApiServices.offChain.withdrawal.storeWithdrawal({
+            ...withdrawal,
+            fee: new BigNumber(fee || '0.001').toString(),
+          })
+
+          try {
+            return {
+              ...(await ApiServices.offChain.withdrawal.broadcastBlockchainTransaction({
```

Comment:
you are not broadcasting the signed transaction here, only preparin it. Also, if it is KMS call - it should call API directly